pkg/mover/tar.go

```diff
@@ -34,6 +58,8 @@ func tarDirectory(rootPath, tarFile string) error {
 			Name:    name,
 			Mode:    int64(info.Mode()),
 			ModTime: info.ModTime(),
+			Uname:   uname(info),
```

Comment:
Out of curiosity, why do we need to manually figure out the user, shouldn't archive.tar default to the user that's running the program?

Reply:
The **stdlib** low level code does not set anything, we need to explicitly set it.

I need to double check but I believe gnu tar would preserve file permissions, ownership and all that from the files, not the executing user.

Reply:
> gnu tar would preserve file permissions,

and wouldn't that be enough?

Reply:
I really think this is better than having `0/0` showing up in a `tar tvf`:
~~~bash
$ tar tvf /tmp/test-tar-dir-2133827212/test.tar 
-rwxr-x--- 0/0              19 2021-11-24 17:18 .hiddenfile
drwxr-x--- 0/0               0 2021-11-24 17:18 dir1
drwxr-x--- 0/0               0 2021-11-24 17:18 dir1/dir2
...
~~~

Reply:
I am not sure why that happens to you, I just did a quick test and the user/group is preserved 

https://gist.github.com/migmartri/06fcb75e4f9d88e1e310472ca5d672d2

Reply:
> What code did you use to create /tmp/test-tar-dir-2133827212/test.tar?
The same code without the figuring out, which is basically your code. So it does not make much sense.

I will try to remove the code again and compare with your gist above. But will do is separately.

Reply:
> I will try to remove the code again and compare with your gist above. But will do is separately.

please don't merge code that will be potentially deprecated from day one. Let's double check that the user/group code is necessary and remove it otherwise

Reply:
btw my code didn't use "real files" so I am not sure if the discrepancy is related to that. In any case, what user/group do the bundle files have when created prior to getting tared?

Reply:
> please don't merge code that will be potentially deprecated from day one. Let's double check that the user/group code is necessary and remove it otherwise. Thanks!

I guess that you decided to merge it anyways. Let's please revisit this, thanks